conduct.md

```diff
@@ -40,25 +40,34 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating.
+
+- Constructively criticize others.
```

Comment:
Please provide examples/descriptions of how you should constructively criticize students' participation. Additionally, please elaborate on how this will be beneficial towards acceptable conduct